Makefile

```
@@ -24,3 +24,26 @@ build-push-deploy-operator:
 
 push-helm-chart:
 	./ci/push-helm-chart.sh
+
+helm-upgrade:
+	helm upgrade --install tailing-sidecar \
+	--namespace tailing-sidecar-system \
+	--create-namespace \
+	helm/tailing-sidecar-operator
```

```suggestion
	helm upgrade --install tailing-sidecar \
		--namespace tailing-sidecar-system \
		--create-namespace \
		helm/tailing-sidecar-operator
```

same for other targets